recipes/poppler/all/conanfile.py

```diff
@@ -132,125 +126,119 @@ def _minimum_compilers_version(self):
         }
 
     def validate(self):
-        if self.options.fontconfiguration == "win32" and self.settings.os != "Windows":
+        if self.info.options.fontconfiguration == "win32" and self.settings.os != "Windows":
             raise ConanInvalidConfiguration("'win32' option of fontconfig is only available on Windows")
 
         # C++ standard required
-        if self.settings.compiler.get_safe("cppstd"):
-            tools.check_min_cppstd(self, 14)
+        if self.info.settings.compiler.get_safe("cppstd"):
+            check_min_cppstd(self, 14)
 
         minimum_version = self._minimum_compilers_version.get(str(self.settings.compiler), False)
         if not minimum_version:
             self.output.warn("C++14 support required. Your compiler is unknown. Assuming it supports C++14.")
-        elif tools.Version(self.settings.compiler.version) < minimum_version:
+        elif Version(self.info.settings.compiler.version) < minimum_version:
             raise ConanInvalidConfiguration("C++14 support required, which your compiler does not support.")
 
-        if self.options.with_nss:
+        if self.info.options.with_nss:
             # FIXME: missing nss recipe
             raise ConanInvalidConfiguration("nss is not (yet) available on cci")
 
     def build_requirements(self):
-        self.build_requires("pkgconf/1.7.4")
+        self.tool_requires("pkgconf/1.9.3")
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version],
-                  destination=self._source_subfolder, strip_root=True)
+        get(self, **self.conan_data["sources"][self.version],
+                  destination=self.source_folder, strip_root=True)
 
     @property
     def _dct_decoder(self):
-        if self.options.with_libjpeg == False:
-            return "none"
-        else:
+        if self.options.with_libjpeg:
             return str(self.options.with_libjpeg)
 
+        return "none"
+
     @property
     def _cppstd_required(self):
-        if self.options.with_qt and tools.Version(self.deps_cpp_info["qt"].version).major == "6":
+        if self.options.with_qt and Version(self.deps_cpp_info["qt"].version).major == "6":
             return 17
+
+        return 14
+
+    def _patch_sources(self):
+        apply_conandata_patches(self)
+
+        copy(self, "FindOpenJPEG.cmake", self.export_sources_folder, os.path.join(self.source_folder, "cmake", "modules"))
+
+        if Version(self.version) < "22.07.0":
+            replace_in_file(self, os.path.join(self.source_folder, "CMakeLists.txt"),
+                    "FREETYPE_INCLUDE_DIRS",
+                    "Freetype_INCLUDE_DIRS")
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["CMAKE_CXX_STANDARD"] = self._cppstd_required
```

Comment:
Should be handled by the toolchain 🤔 

Maybe the default 1.x it's not defined need to be handled here?

`if settings.compiler.get_safe("sppstd") == None: The set it` tye of thing?